### PR TITLE
Minor transition and hover effects added

### PR DESCRIPTION
**Pull request by Shridhar-dev**

### CODE REVIEW

File: src/components/card/horizontal-resource-card.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const HorizontalResourceCard: React.FC<any> = ({
   className = 'border-none my-4',
   ...props
 }) => {
-  className = `${className} flex sm:flex-row flex-col sm:space-x-5 space-x-0 sm:space-y-0 space-y-5 items-center sm:text-left text-center`
+  className = `${className} flex sm:flex-row transition duration-300 hover:shadow-md transform hover:-translate-y-1 flex-col sm:space-x-5 space-x-0 sm:space-y-0 space-y-5 items-center sm:text-left text-center`
```
Comment (DrShpongle) [inline src/components/card/horizontal-resource-card.tsx:22]: Only the nitpick is that the `transform` class is no longer necessary to "enable" their respective set of composable utilities: https://github.com/tailwindlabs/tailwindcss/releases/tag/v2.2.0

**Comment by DrShpongle** [comment]
Looks good to me 🙌

**Comment by Shridhar-dev** [comment]
Ohk, will remove the `transform` class

**Review by DrShpongle — APPROVED** [review]

**Comment by vojtaholik** [comment]
thanks @Shridhar-dev! btw screenshots or gifs are always appreciated, especially for design updates like this one.

**Comment by Shridhar-dev** [comment]
Welcome 😄, will take care of the screenshot part next time 👍

**Comment by vojtaholik** [comment]
Hey @Shridhar-dev, we appreciate your contribution, but we’ve decided to close this PR since we’re planning more UI updates around Card components in near future. We’re happy to chat about suggestions you might have in [Discussions](https://github.com/eggheadio/egghead-next/discussions), so please feel free to share them there.

Thanks again!

**Comment by Shridhar-dev** [comment]
Yup, sure!

**Pull request closed without merging**